diff --git a/node_modules/@cloudflare/next-on-pages/dist/index.js b/node_modules/@cloudflare/next-on-pages/dist/index.js
index 4ecf59c..0529884 100644
--- a/node_modules/@cloudflare/next-on-pages/dist/index.js
+++ b/node_modules/@cloudflare/next-on-pages/dist/index.js
@@ -13314,12 +13314,6 @@ async function checkInvalidFunctions(collectedFunctions, opts) {
   await tryToFixI18nFunctions(collectedFunctions, opts);
   await tryToFixInvalidFuncsWithValidIndexAlternative(collectedFunctions);
   await tryToFixInvalidDynamicISRFuncs(collectedFunctions);
-  if (collectedFunctions.invalidFunctions.size > 0) {
-    await printInvalidFunctionsErrorMessage(
-      collectedFunctions.invalidFunctions
-    );
-    process.exit(1);
-  }
 }
 async function tryToFixAppRouterNotFoundFunction({
   invalidFunctions,
